tests/test_integration_history.py

```diff
@@ -0,0 +1,308 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+from flask import json
+
+from . import util
+
+
+class HistoryTest(util.LoRATestCase):
```

Comment:
Our previous history tests ran each test as its own class — in order to ensure that they started out with a clean slate. Do these tests still pass if you run them in reverse order?

Reply:
They operate on different objects, so they're safe.
Runnings each of them in separate classes is still a good idea, though. I'll fix that.

Reply:
As an optimisation, we don't actually create a full LoRA instance for test, but rather for each class. Then, we simply iterate over all relevant objects between runs and `DELETE` them. This is fine, as long as we don't happen to look at the registration history of each object — which is true for everything other than our history support. Hence my question…